crates/execution/src/matching_engine/engine.rs

```diff
@@ -1106,45 +1106,129 @@ impl OrderMatchingEngine {
         self.iterate_orders(timestamp_ns, &orders_ask);
     }
 
+    fn maybe_activate_trailing_stop(
+        &mut self,
+        order: &mut OrderAny,
+        bid: Option<Price>,
+        ask: Option<Price>,
+    ) -> bool {
+        match order {
+            OrderAny::TrailingStopMarket(inner) => {
+                if inner.is_activated {
+                    return true;
+                }
+
+                if inner.activation_price.is_none() {
+                    let px = match inner.order_side() {
+                        OrderSide::Buy => ask,
+                        OrderSide::Sell => bid,
+                        _ => None,
+                    };
+                    if let Some(p) = px {
+                        inner.activation_price = Some(p);
+                        inner.set_activated();
+                        if let Err(e) = self.cache.borrow_mut().update_order(order) {
+                            log::error!("Failed to update order: {e}");
+                        }
+                        return true;
+                    }
+                    return false;
+                }
+
+                let ap = inner.activation_price.unwrap();
```

Comment:
`activation_price` as a variable name would be clearer and consistent here

Reply:
done, thx